lib/utils.pm

```diff
@@ -1908,22 +1909,7 @@ sub install_patterns {
         }
         # if pattern is common-criteria and PATTERNS is all, skip, poo#73645
         next if (($pt =~ /common-criteria/) && check_var('PATTERNS', 'all'));
-        my $ret = zypper_call("in -t pattern $pt", exitcode => [0, 4], timeout => 1800);
-        if ($ret == 4) {
-            if (($pt =~ /CFEngine/) && is_sle('>=12') && is_sle('<15')) {
-                record_soft_failure 'bsc#1175704 pattern:CFEngine-12-8.1.s390x requires patterns-adv-sys-mgmt-CFEngine, but this requirement cannot be provided';
-                my $para = '';
-                $para = '--force-resolution' if get_var('FORCE_DEPS');
-                zypper_call("in $para -t pattern $pt", timeout => 1800, exitcode => [0, 102, 103]);
-            }
-            else {
-                script_run('tac /var/log/zypper.log | grep -F -m1 -B10000 "Hi, me zypper" | tac | grep \'Exception.cc\' > /tmp/zlog.txt');
-                my $msg = "'zypper -n in -t pattern $pt failed with code $ret";
-                $msg .= "\n\nRelated zypper logs:\n";
-                $msg .= script_output('cat /tmp/zlog.txt');
-                die $msg;
-            }
-        }
+        zypper_call("in -t pattern $pt", timeout => 1800);
```

Comment:
Lemon, why this install process take 1800s? can you add some comments here?

Reply:
In fact, I haven't changed here, I think it is for some pattern need wait long time, but I haven't met such situation anyway.